WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -446,9 +446,16 @@ && getIntent().getExtras().getBoolean(ARG_CONTINUE_JETPACK_CONNECT, false)) {
             onSetPromptReminderClick(getIntent().getIntExtra(ARG_OPEN_BLOGGING_REMINDERS, 0));
         }
 
+        int selectedSiteLocalId = mSelectedSiteRepository.getSelectedSiteLocalId(true);
```

Comment:
What is this required for?

WPMainActivity initializes selected site (initSelectedSite() L:1508)
There's also a method getSelectedSite() (L:1490)

if really required then, we could do like 

        if (!mSelectedSiteRepository.hasSelectedSite()) {
            initSelectedSite();
        }

      getSelectedSite() != null

Reply:
Updated b3044b1be1302bcbea3bc2cac90cc657b5da1435